### ISSUE TITLE
Adjust component library to have media metacomponent showing correctly on geeks website

### ISSUE THREAD

**Issue by akramsarkis** [Geeks-Solutions/vue-components#14]
- [x] Extract the css from tailwind that are required by the component and embed the css inside the library so it gets styled properly regardless of what css library the host project is using

**Acceptance testing**

- [x] Make sure that the media metacomponent is showing correctly and working fine on geeks website, this will be tested with issue 162

**Comment by akramsarkis** [comment]
Estimated 1h30

**Comment by mahfouzhseineweev** [comment]
The issue took more time than expected because of the following:

We needed a way to have Tailwind css classes of the media meta component working without having to install Tailwind css library because it is conflicting with other CSS libraries like bootstrap. So I wanted to update all Tailwind CSS classes used with other classes that point to plan css style. But then I realized that I will have to do that to every class in all components that medias are using and not just the 3 media meta components which will take a lot of time to be done and tested.

So instead of that I went to try an option provided by the Tailwind css plugin which extract Tailwindcss classes into a plain css file using this command `npx tailwindcss build input.css -o output.css`. Then I took the file, I added it to the vue components library but without using it by default.

Then from the nuxt-sections library that has the vue components installed I perform wether or not Tailwind css is installed on the host project and if it is not I include the output css that I have added to the vue components library and so all css styles will take effect using this file.
